src/accountScreen/accountScreen.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import './accountScreen.css'
+import SearchBar from 'material-ui-search-bar'
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+
+
+const ChatCard = function(props) {
+  if (props.selected) {
+    return (
+      <Card style={{
+        backgroundColor:'#cacaca'}}>
+        <CardHeader
+          title={props.title}
+          avatar={props.avatar}
+        />
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+
+  else {
+    return (
+      <Card>
+        <CardHeader
+          title={props.title}
+          avatar={props.avatar}/>
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+  
+};
+
+const ChatBubbleRight = function(props) {
+  return (
+    <div className='bubbleContainerRight'>
+      <div className='bubbleRight'>
+        <Card>
+          <CardText>
+            {props.text}
+          </CardText>
+        </Card>
+      </div>
+    </div>
+  
+  );
+}
+
+const ChatBubbleLeft = function(props) {
+  return (
+    <div className= 'bubbleContainerLeft'>
+      <div className='bubbleLeft'>
+        <Card>
+          <CardText>
+            {props.text}
+          </CardText>
+        </Card>
+      </div>
+    </div>
+  
+  );
+}
+
+
+class AccountScreen extends Component {
+
+
+
+  render() {
+    return (
+      <div>
+        <div className='rectangle4'>

```

Comment:
Use proper naming conventions. Do not call css classes "rectangle4"

Reply:
Sorry about that. The designer used dash styling for the css classes (so it was rectangle-4). I was told to use one style convention across my code, so I used camel casing (rectangle4). Is this wrong? Should I use dash casing for css specifically?

Reply:
My comment was not regarding naming rectangle-4 or rectangle4 (ie camel case vs dashes). My problem is that you are naming the class "rectangle4". Why is it #4. Where are 1, 2 and 3... what is the significance of the div being rectangle4? Surely there must be a more descriptive name you can give the div class

Reply:
Alright understood. notified team and pushed changes